### DISCUSSION TITLE
为什么PCL的mod搜索提示连不上curseforge，但电脑浏览器可以访问www.curseforge.com

### DISCUSSION THREAD

**Discussion by ZerkyLiu** [Meloong-Git/PCL#3839] [提问] [answered]
（如题）为什么PCL的mod搜索提示连不上curseforge，但电脑浏览器可以访问www.curseforge.com。

**Comment by ZerkyLiu** [comment]
这种情况每次PCL提示连不上curseforge就会发生。我甚至可以在curseforge上下载mod，但PCL就是提示连不上

**Comment by Deep-Dark-Forest** [comment]
发下截图？

**Comment by zkitefly** [comment]
https://api.curseforge.com/ 能访问吗

**Comment by wuliaodexiaoluo** [comment]
能连上是因为浏览器的超时时间比 PCL长（大概 75秒）
以及可以访问网站并不等于可以请求 api，curseforge 的 website api 和 curseforge api是分开的

**Comment by allMagicNB** [comment, marked as answer]
#3227
